editoast/editoast_derive/src/modelv2/codegen/count_impl.rs

```diff
@@ -9,11 +9,19 @@ pub(crate) struct CountImpl {
 impl ToTokens for CountImpl {
     fn to_tokens(&self, tokens: &mut proc_macro2::TokenStream) {
         let Self { model, table_mod } = self;
+        let span_name = format!("model:list<{}>", model);
 
         tokens.extend(quote! {
             #[automatically_derived]
             #[async_trait::async_trait]
             impl crate::modelsv2::prelude::Count for #model {
+                #[tracing::instrument(name = #span_name, skip_all, ret, err, fields(
+                    nb_filters = settings.filters.len(),
+                    nb_sorts = settings.sorts.len(),
```

Comment:
Sorting is irrelevant for counting, but some sorts might be there if the settings are used elsewhere, you can remove it if you want but I'm fine leaving it like this.

Reply:
Removed.